src/main/java/com/alkemy/ong/model/Organizations.java

```diff
@@ -0,0 +1,59 @@
+package com.alkemy.ong.model;
+
+import java.time.LocalDate;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
```

Comment:
Please delete the unused imports:
import javax.persistence.Table;
import org.hibernate.annotations.CreationTimestamp;
import org.hibernate.annotations.UpdateTimestamp;
import org.springframework.lang.NonNull;
import lombok.Getter;
import lombok.Setter;